hnn_core/viz.py

```diff
@@ -849,7 +865,8 @@ def plot_cell_morphology(cell, ax, show=True):
             xs.append(pt[0] + dx)
             ys.append(pt[1] + dz)
             zs.append(pt[2] + dy)
-        ax.plot(xs, ys, zs, 'b-', linewidth=linewidth)
```

Comment:
The `'b-'` is redundant now since the color `'b'` is explicitly set when `None` is passed to `color`